yt_dlp/extractor/tennistv.py

```diff
@@ -31,84 +34,126 @@ class TennisTVIE(InfoExtractor):
     }
     _NETRC_MACHINE = 'tennistv'
 
-    def _login(self):
-        username, password = self._get_login_info()
-        if not username or not password:
-            raise ExtractorError('No login info available, needed for using %s.' % self.IE_NAME, expected=True)
-
-        login_form = {
-            'Email': username,
-            'Password': password,
-        }
-        login_json = json.dumps(login_form).encode('utf-8')
-        headers = {
-            'content-type': 'application/json',
-            'Referer': 'https://www.tennistv.com/login',
-            'Origin': 'https://www.tennistv.com',
-        }
+    ACCESS_TOKEN = None
+    REFRESH_TOKEN = None
+    PARTNER_ID = 3001482
+    _FORMAT_URL = 'https://open.http.mp.streamamg.com/p/{partner}/sp/{partner}00/playManifest/entryId/{entry}/format/applehttp/protocol/https/a.m3u8?ks={session}'
+    headers = {
+        'origin': 'https://www.tennistv.com',
+        'referer': 'https://www.tennistv.com/',
+        'content-Type': 'application/x-www-form-urlencoded'
+    }
 
-        login_result = self._download_json(
-            'https://www.tennistv.com/api/users/v1/login', None,
-            note='Logging in',
-            errnote='Login failed (wrong password?)',
-            headers=headers,
-            data=login_json)
+    def _login(self):
```

Comment:
See the new documentation on adding login support. The merge conflict should disappear once this is fixed
https://github.com/yt-dlp/yt-dlp/blob/ae6a1b95857c7ba1392e37be99b9ee2b66b14b1d/yt_dlp/extractor/common.py#L437-L443